keyboards/mino_plus/hotswap/rules.mk

```diff
@@ -0,0 +1,3 @@
+OLED_ENABLE = yes
```

Comment:
If OLEDs are enabled at the keyboard level, there needs to exist some basic functionality for them at keyboard level as well, in the `hotswap.c` file using the `oled_update_kb()` function.

Reply:
I will move this so it will be enabled at the global level and will leave the OLED code in `mino_plus.c`